src/main/java/com/codessquad/qna/web/domain/Answer.java

```diff
@@ -77,12 +70,8 @@ public void setQuestion(Question question) {
         this.question = question;
     }
 
-    public LocalDateTime getReportingDateTime() {
-        return reportingDateTime;
-    }
-
-    public void setReportingDateTime(LocalDateTime reportingDateTime) {
-        this.reportingDateTime = reportingDateTime;
+    public Boolean getDeleted() {
```

Comment:
```suggestion
    public boolean isDeleted() {
```

이 편이 훨씬 일반적인 네이밍 입니다.

Reply:
가급적이면 인텔리제이의 getter 생성기를 이용해주세요.

Reply:
그렇군요! 주의하도록 하겠습니다! 수정해서 다시 커밋할께요!